apps/fabric-website/src/pages/HomePage/HomePage.tsx

```diff
@@ -14,11 +14,21 @@ const reactPackageData = require('office-ui-fabric-react/package.json');
 const versionLinkId = getId('versionLink');
 
 // Update as new Fabric versions are released
-const fabricVersionOptions: IContextualMenuItem[] = [{ key: 'A', text: 'Fabric 6', data: '6' }, { key: 'B', text: 'Fabric 5', data: '5' }];
+const fabricVersionOptions: IContextualMenuItem[] = [
```

Comment:
Is this just a prettier change?

Reply:
It's my own formatting change, it was a bit hard to read on a single line